Client/mods/deathmatch/logic/luadefs/CLuaEngineDefs.cpp

```diff
@@ -928,6 +930,40 @@ int CLuaEngineDefs::EngineGetVisibleTextureNames(lua_State* luaVM)
     return 1;
 }
 
+int CLuaEngineDefs::EngineGetModelTextures(lua_State* luaVM)
+{
+    //  table engineGetModelTextures ( string modelName )
+    SString strModelName;
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadString(strModelName, "");
+
+    if (argStream.HasErrors())
+        return luaL_error(luaVM, argStream.GetFullErrorMessage());
+
+    ushort usModelID = CModelNames::ResolveModelID(strModelName);
+    if (usModelID != INVALID_MODEL_ID)
```

Comment:
I'd prefer `if (usModelID == INVALID_MODEL_ID) return 1;` here to reduce indentation depth and to keep consistent with the `HasErrors` call above.